.github/workflows/data_library_single.yml

```diff
@@ -28,8 +28,7 @@ on:
 jobs:
   dataloading:
     runs-on: ubuntu-22.04
-    container:
-      image: osgeo/gdal:ubuntu-small-3.6.1
+    container: python:3.11-slim
```

Comment:
Still running from a real base image rather than one of ours to be able to test local changes. Didn't seem worth having an additional image with just the python requirements installed. `./library.sh setup` below takes care of the container setup